bin/alluxio-secondary-masters.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
@peisun1115 how do you feel about using simply `secondary` instead of `secondary-master` and `secondaries` instead of `secondary-masters`?